src/main/java/com/odeyalo/sonata/connect/repository/storage/support/InMemory2PersistablePlayerStateConverter.java

```diff
@@ -16,6 +16,7 @@ public PersistablePlayerState convertTo(InMemoryPlayerState type) {
                 .shuffleState(type.getShuffleState())
                 .progressMs(type.getProgressMs())
                 .repeatState(type.getRepeatState())
+                .user(type.getUser())
```

Comment:
I need to update this file every time I add new field to entity, should it be like that? Maybe rewrite it with MapStruct? I think it should be in one of the future releases